101-cognitive-services-Computer-vision-API/azuredeploy.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
+      "defaultValue": "computervision",
```

Comment:
if this is globally unique, remove the defaultValue

Reply:
Hi Brian,
this has not to be unique